app/views/smart_proxies/logs/_modal.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="modal fade" id="backtrace-modal" tabindex="-1" role="dialog" aria-labelledby="bt-modal-label">
```

Comment:
We do have wider modals in some places, host interfaces is the first that comes to mind.
The new layout does look better though. I think you could fit the first three lines (time and level) in one row possibly, leaving more space for the details. Also, possibly add a slight border to the overflowing areas so it would be clearer what the scroll bars are related to.